### PR TITLE
add kops-bazel jobs

### PR DESCRIPTION
**Pull request by BenTheElder**
also "fix" test-infra-bazel config.json (set the same scenario we're actually using via prow config.yaml)

/area jobs

### CODE REVIEW

**Comment by BenTheElder** [comment]
I left remote caching off on the build for now, I want to muck with upgrading the server to more space first, and so far unit tests cache a bit better anyhow.

**Comment by BenTheElder** [comment]
also set these to skip_report for now so we can make sure they work

File: jobs/config.json [hunk 1]
```diff
@@ -11173,6 +11173,22 @@
       "sig-node"
     ]
   },
+  "pull-kops-bazel-build": {
+    "_comment": "NOTE: args for this are in prow/config.yaml after the `--` !",
+    "args": [],
```
Comment (chrislovecnm) [inline jobs/config.json]: we do not need args?

Comment (BenTheElder) [reply jobs/config.json]: nope, they're inlined to prow/config.yaml. someday I will delete this file.
xref #4794

**Comment by BenTheElder** [comment]
forgot to add these to testgrid, one sec.

**Comment by BenTheElder** [comment]
PTAL, fixed testgrid

File: jobs/config.json [hunk 2]
```diff
@@ -11173,6 +11173,22 @@
...
+    "scenario": "kubernetes_execute_bazel",
+    "sigOwners": [
+      "sig-cluster-lifecycle"
```
Comment (krzyzacy) [inline jobs/config.json]: sig aws

File: jobs/config.json (hunk 1 above)
Comment (krzyzacy) [reply jobs/config.json]: we probably don't need a config.json entry in this case?

File: jobs/config.json (hunk 2 above)
Comment (BenTheElder) [reply jobs/config.json]: the other kops jobs are sig-cluster-lifecycle. I think only kops-aws is aws...

File: jobs/config.json (hunk 1 above)
Comment (BenTheElder) [reply jobs/config.json]: there are presubmits that blow up without one

**Comment by BenTheElder** [comment]
https://github.com/kubernetes/test-infra/pull/6664#issuecomment-363321116
image
we still need to make that message better :joy: @cjwagner

**Comment by cjwagner** [comment]
Agreed, adding markdown formatting is probably sufficient?

**Pull request merged**